whetstone.py

```diff
@@ -0,0 +1,53 @@
+import os
+import requests
+import base64
+
+
+class CredentialError(Exception):
+    def __init__(self):
+        self.message = "Authorization Failed: Verify CLIENT_ID and CLIENT_SECRET are set in the environment"
+        super().__init__(self.message)
+
+
+class Whetstone:
+    def __init__(self, qa=False):
+        self.client_id = os.getenv("CLIENT_ID")
+        self.client_secret = os.getenv("CLIENT_SECRET")
+        if qa:
+            subdomain = "api-qa"
+        else:
+            subdomain = "api"
```

Comment:
This is a perfect spot for a ternary function if you want to use it.
`subdomain = "api-qa" if qa else "api"`

Reply:
I like this. I will change it in #2.